contrib/babelfishpg_tsql/src/pltsql_utils.c

```diff
@@ -1791,3 +1791,78 @@ List
 
 	return stmt_list;
 }
+
+/*
+ * get_index_name
+ *		Get the index name of a relation specified by OID
+ */
+char
+*get_index_name(Oid relid)
+{
+	Relation		relation;
+	List			*indexoidlist;
+	Oid				indexOid;
+	char			*index_name;
+
+	relation = RelationIdGetRelation(relid);
+	indexoidlist = RelationGetIndexList(relation);
+	if (list_length(indexoidlist) == 0)
+	{
+		RelationClose(relation);
+		list_free(indexoidlist);
+		return NULL;
+	}
+	indexOid = linitial_oid(indexoidlist);
```

Comment:
Because there can be only one fulltext index per table acc. to SQL Server docs. We get a list of indexes, so we assume we get the fulltext index in the first item only.